### DISCUSSION TITLE
Use of the Toolkit with VB

### DISCUSSION THREAD

**Discussion by SezMe** [CommunityToolkit/dotnet#600] [Q&A]
I'd like to use the Toolkit with VB but it doesn't seem possible.  Is that correct?

**Comment by Iridium-IO** [comment]
It works perfectly fine in VB.NET, the only issue I've run into so far is that attributes (e.g. `<RelayCommand>`, `<ObservableProperty>`) do not work and you can't use these shortcut methods. Not a big deal though.

**Reply by SezMe** [reply]
Yes, it does ... with the restrictions you cited.  It would be useful the for VB programmer that those restrictions be explicitly stated in the overview, at least.  It would have saved me a boatload of time.

**Reply by Iridium-IO** [reply]
You and me both lol, I did submit a request for that information to be added, so hopefully they add it before the next guy wastes hours on it. 

In the meantime, I'm just using [Fody/PropertyChanged](https://github.com/Fody/PropertyChanged) to avoid having to call it manually on everything
